app/src/main/java/com/example/arc_exapmle/NoteUI.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.arc_exapmle
+
+class NoteUI(noteEntity: NoteEntity?) {
+    internal var title: String? = null
+    internal var description: String? = null
+    internal var priority: Int? = null
+
+    init {
+        title = noteEntity?.title
+        description = noteEntity?.description
+        priority = noteEntity?.priority
+    }
```

Comment:
in karo bayad biron az in class anjam bedi
in class nabayad niazi be NoteEntity dashte bashe.
bayad faghat parameter haye vorodi sho begire.
dar zemn behtare id ham pas bedi

Reply:
anjam shod